cmd/serviceusers/update.go

```diff
@@ -0,0 +1,169 @@
+package serviceusers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/appgate/sdpctl/pkg/docs"
+	"github.com/appgate/sdpctl/pkg/factory"
+	"github.com/appgate/sdpctl/pkg/filesystem"
+	"github.com/appgate/sdpctl/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+func NewServiceUsersUpdateCMD(f *factory.Factory) *cobra.Command {
+	opts := ServiceUsersOptions{
+		Config: f.Config,
+		API:    f.ServiceUsers,
+		Out:    f.IOOutWriter,
+	}
+	cmd := &cobra.Command{
+		Use:     "update [id] [args...]",
+		Short:   docs.ServiceUsersUpdate.Short,
+		Long:    docs.ServiceUsersUpdate.Long,
+		Example: docs.ServiceUsersUpdate.ExampleString(),
+		Aliases: []string{"edit", "set"},
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return fmt.Errorf("not enough arguments")
+			}
+			if !util.IsUUID(args[0]) {
+				return fmt.Errorf(InvalidUUIDError)
+			}
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I think you can reduce indentation level significantly here if you refactor and return earlier(/and or extract funcs)